srcpkgs/flyctl/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'flyctl'
+pkgname=flyctl
+version=0.0.351
+revision=1
+build_style=go
+go_import_path="github.com/superfly/flyctl"
+go_ldflags="
+ -X github.com/superfly/flyctl/internal/buildinfo.environment=production
+ -X github.com/superfly/flyctl/internal/buildinfo.version=$version
+ -X github.com/superfly/flyctl/internal/buildinfo.commit=df8c42a
+ -X github.com/superfly/flyctl/internal/buildinfo.buildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
```

Comment:
Should be a constant build date for reproducibility I think...

Reply:
yes, something like this:
https://github.com/void-linux/void-packages/blob/f64ff867bab6fcce90c2ef6e7c04865d2b2b2032/srcpkgs/chezmoi/template#L16-L22